src/components/createElementComponent.test.tsx

```diff
@@ -289,6 +253,81 @@ describe('createElementComponent', () => {
       );
     });
 
+    it('adds an event handlers to an Element', () => {
+      const mockHandler = jest.fn();
+      render(
+        <Elements stripe={mockStripe}>
+          <CardElement onChange={mockHandler} />
+        </Elements>
+      );
+
+      const changeEventMock = Symbol('change');
+      simulateEvent('change', changeEventMock);
+      expect(mockHandler).toHaveBeenCalledWith(changeEventMock);
+    });
```

Comment:
To test the failure scenario you were able to create repro steps for, maybe try creating a wrapper that simulates your repro steps? Can't remember the details exactly, but something like this?

```
      const mockHandler = jest.fn();
      const Wrapper = () => {
        const compoent =  useMemo(() => <CardElement onChange={mockHandler} />, []);
        return component;
      }
      
      render(
        <Elements stripe={mockStripe}>
          <Wrapper />
        </Elements>
      );

      const changeEventMock = Symbol('change');
      simulateEvent('change', changeEventMock);
      expect(mockHandler).toHaveBeenCalledWith(changeEventMock);
```

Reply:
I tried to do `useMemo` like the repro but unfortunately didn't have much luck. However, along this same idea of copying the repro, I got a test together that leaves `elements` undefined, preventing it from creating an element on the first render. This test fails on v1.16.1 but passes in 1.16.2 & here!